### PR TITLE
Add aspnet:2.1 and aspnet:2.2 image streams

### PR DESCRIPTION
**Pull request by tmds**
These image stream definitions use a Docker strategy to add
ASP.NET Core to the s2i dotnet image streams.

The resulting images have faster build times because they
include the NuGet packages for ASP.NET Core which are otherwise
downloaded from a NuGet repository (e.g. nuget.org).

The images have a change trigger for the dotnet imagestreams.
They are automatically re-built when those images are updated
with security patches.

We are not providing these images directly, because we can only
distribute images which were completely built from source.

### CODE REVIEW

**Comment by tmds** [comment]
Some numbers building s2i-dotnetcore-ex.
Using dotnet: 43s to 1m30, avg 47s
Using aspnet: 23s to 31s, avg 24s

File: README.md [hunk 1]
```diff
@@ -124,6 +124,11 @@ The dotnet-pgsql-persistent creates a .NET Core service with a PostgreSQL backen
 variables of the s2i-dotnetcore builder and variables to setup the database. The database connection information is passed to the
 .NET application via the `ConnectionString` environment variable.
 
+**aspnet-2.x.json**
+
+Provides `aspnet:2.1` and `aspnet:2.2` image streams that are built on top of `dotnet:2.1` and `dotnet:2.2` using a `Docker` strategy and include
```
Comment (omajid) [inline README.md]: Will this work with any of the dotnet:2.1 tags? Whether they are RHEL 7, CentOS 7, or RHEL 8 based?

File: README.md [hunk 2]
```diff
@@ -124,6 +124,11 @@ The dotnet-pgsql-persistent creates a .NET Core service with a PostgreSQL backen
...
+ASP.NET Core NuGet packages. Using these streams results in improved build speed, because the NuGet packages no longer need to be downloaded.
```
Comment (omajid) [inline README.md]: Do you plan to add any tests to make sure this imagestream is exercised with future releases too?

Comment (omajid) [inline README.md]: Do we need to add any documentation/examples to show users how to make use of this?

**Comment by omajid** [comment]
This is a really great workaround!

File: README.md (hunk 1 above)
Comment (tmds) [reply README.md]: yes

File: README.md (hunk 2 above)
Comment (tmds) [reply README.md]: No, we haven't automated any imagestream/template testing (https://github.com/redhat-developer/s2i-dotnetcore/issues/171).

Comment (tmds) [reply README.md]: I've extended this section, ptal.

**Comment by tmds** [comment]
> This is a really great workaround!

Thanks!

File: README.md (hunk 2 above)
Comment (omajid) [reply README.md]: Looks good. Thanks!

**Comment by tmds** [comment]
Thank you for reviewing @omajid.

**Pull request merged**